tensorboard/plugins/graph/graphs_plugin.py

```diff
@@ -108,10 +139,25 @@ def add_row_item(run, tag=None):
 
     return result
 
-  def graph_impl(self, run, limit_attr_size=None, large_attrs_key=None):
+  def graph_impl(self, run, tag, conceptual, limit_attr_size=None, large_attrs_key=None):
     """Result of the form `(body, mime_type)`, or `None` if no graph exists."""
     try:
-      graph = self._multiplexer.Graph(run)
+      if conceptual:
+        tensor_events = self._multiplexer.Tensors(run, tag)
+        # Take the first event if there are multiple events written from different
+        # steps.
+        keras_model_config = json.loads(tensor_events[0].tensor_proto.string_val[0])
```

Comment:
I hope to approach this holistically later since our exception handling on our backend/frontend generally lack. As is, it will result in HTTP 500 with some logs in the console.